libs/api/api_internal.h

```diff
@@ -0,0 +1,27 @@
+/*
+ *  Copyright (c) Microsoft Corporation
+ *  SPDX-License-Identifier: MIT
+ */
+
+#pragma once
+
+#include <spec_type_descriptors.hpp>
+#include "ebpf_windows.h"
```

Comment:
```suggestion
#include "ebpf_windows.h"
#include "spec_type_descriptors.hpp"
```
only use `<>` for system files